src/clients/api/queries/getMarkets.ts

```diff
@@ -29,11 +29,27 @@ const getMarkets = async (): Promise<IGetMarketsOutput> => {
   if (response && response.data && response.data.data) {
     dailyVenus = new BigNumber(response.data.data.dailyVenus);
     markets = Object.keys(VBEP_TOKENS)
-      .map(item =>
-        response.data?.data.markets.find(
+      .map(item => {
+        const activeMarket = response.data?.data.markets.find(
           (market: Market) => market.underlyingSymbol.toLowerCase() === item.toLowerCase(),
-        ),
-      )
+        );
+        if (activeMarket) {
+          activeMarket.id = activeMarket.underlyingSymbol.toLowerCase();
+          activeMarket.tokenPrice = new BigNumber(activeMarket.tokenPrice);
+          activeMarket.liquidity = new BigNumber(activeMarket.liquidity);
+          activeMarket.borrowVenusApy = new BigNumber(activeMarket.borrowVenusApy);
+          activeMarket.borrowApy = new BigNumber(activeMarket.borrowApy);
+          activeMarket.supplyVenusApy = new BigNumber(activeMarket.supplyVenusApy);
+          activeMarket.supplyApy = new BigNumber(activeMarket.supplyApy);
+          activeMarket.treasuryTotalBorrowsUsdCents = new BigNumber(
+            activeMarket.totalBorrowsUsd,
+          ).times(100);
+          activeMarket.treasuryTotalSupplyUsdCents = new BigNumber(
+            activeMarket.totalSupplyUsd,
+          ).times(100);
+        }
+        return activeMarket;
+      })
       .filter(notUndefined);
   }
```

Comment:
We should avoid mutating variables like that: if later on the code evolves and `response` is used after this, its content will be mutated which might be unexpected by developers.
We can also handle everything with one loop only:
```typescript
if (response?.data?.data) {
  dailyVenus = new BigNumber(response.data.data.dailyVenus);
  markets = Object.keys(VBEP_TOKENS).reduce<Market[]>((acc, vBepTokenId) => {
    const activeMarket = response.data?.data.markets.find(
      (market: Market) => market.underlyingSymbol.toLowerCase() === vBepTokenId.toLowerCase(),
    );

    if (activeMarket) {
      const formattedActiveMarket: Market = {
        ...activeMarket,
        id: activeMarket.underlyingSymbol.toLowerCase(),
        tokenPrice: new BigNumber(activeMarket.tokenPrice),
        liquidity: new BigNumber(activeMarket.liquidity),
        borrowVenusApy: new BigNumber(activeMarket.borrowVenusApy),
        borrowApy: new BigNumber(activeMarket.borrowApy),
        supplyVenusApy: new BigNumber(activeMarket.supplyVenusApy),
        supplyApy: new BigNumber(activeMarket.supplyApy),
        treasuryTotalBorrowsUsdCents: new BigNumber(activeMarket.totalBorrowsUsd).times(100),
        treasuryTotalSupplyUsdCents: new BigNumber(activeMarket.totalSupplyUsd).times(100),
      };

      return [...acc, formattedActiveMarket];
    }

    return acc;
  }, []);
}
```